docs/serving/services/creating-services.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Creating Knative Services
+type: “docs”
+---
+
+Knative services are used to deploy an application. Knative Services attempt to model a more logical definition of a Service or Application. To achieve this, Services use the concept of Revisions which are each backed by a deployment with 2 Kubernetes Services.   
+
+For more information about Kubernetes Services, see the [Kubernetes documentation](https://kubernetes.io/docs/concepts/services-networking/service/).   
+
+Each Knative Service is defined by a Route and a Configuration, which have the same name as the Service, contained in a YAML file.
+
+## Before you begin
+
+To use Knative Services, you will need:
+* A Kubernetes cluster with [Knative installed](https://knative.dev/docs/install/index.html).
+* [Custom domains](https://knative.dev/docs/serving/using-a-custom-domain/) set up for Knative Services.
+
+## Creating a Knative Service
+To create an application, you need to create a YAML file that defines a Knative Service.
+This YAML file specifies metadata about the application, points to the hosted image of the app and allows the Service to be configured.
+
+This guide uses the [Hello World sample app in Go](https://knative.dev/docs/serving/samples/hello-world/helloworld-go) to demonstrate the structure of a Service YAML file and the basic workflow for deploying an app. These steps can be adapted for your own application if you have an image of it available on Docker Hub, Google Container Registry, or another container image registry.
+
+The Hello World sample app does the following:
+1. Reads an environment variable, `TARGET`, from the configuration .yaml file
+1. Prints `Hello World: \${TARGET}!`. If `TARGET` is not defined, it will print `NOT SPECIFIED`.
+
+### Procedure
+
+1. Create a new file named `service.yaml` containing the following information.
```

Comment:
This ordered list only has one item in it. I think it would make sense to demonstrate applying the .yaml so the service is actually created, which would add another step.